WordPress/Classes/Utility/BuildInformation/FeatureFlag.swift

```diff
@@ -17,7 +17,7 @@ enum FeatureFlag: Int {
         case .jetpackDisconnect:
             return BuildConfiguration.current == .localDeveloper
         case .quickStart:
-            return BuildConfiguration.current == .localDeveloper
+            return true
```

Comment:
This is cool for launch, but can we open an issue to remind us to clean up the Flag later?

Reply:
Yes!